### ISSUE TITLE
No obvious way to "Share Link" on android

### ISSUE THREAD

**Issue by bobpaul** [simplex-chat/simplex-chat#419]
Android app v0.4.2
It says that I can share either a [QR code or a link](https://imgur.com/aFBQ11H.png), but there's [nothing obvious to click on for the link](https://imgur.com/zMRyFIQ.png). I did find if I tap a bunch of times between the QR code and the home button on my phone, it eventually pops open the Android share dialog and I can copy the link to my clipboard.

I think there's probably a button that's rendered off screen. I'm using a Galaxy S10e and haven't enabled any accessibility options (like increased font size)

**Comment by epoberezkin** [comment]
yes, it's below the screen with lower resolution - sorry - @IanRDavies is fixing it!

**Comment by IanRDavies** [comment]
This was fixed in #417 and will be included in the next android release. Thank you for raising the issue!
